subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1319,6 +1321,10 @@ int ull_adv_reset(void)
 {
 	uint8_t handle;
 
+	for (handle = 0U; handle < BT_CTLR_ADV_SET; handle++) {
```

Comment:
accepted this change, but reset of Periodic Advertising is missing and can be added in another PR.